lib/stream.js

```diff
@@ -137,7 +137,7 @@ ObjectDefineProperty(eos, customPromisify, {
 // Backwards-compat with node 0.4.x
 Stream.Stream = Stream;
 
-Stream._isUint8Array = require('internal/util/types').isUint8Array;
+Stream._isArrayBufferView = require('internal/util/types').isArrayBufferView;
```

Comment:
Can you please keep the other method around too? It'll be used by readable-stream